diktat-cli/src/test/resources/test/smoke/src/main/kotlin/Example5Expected.kt

```diff
@@ -1,6 +1,6 @@
 // ;warn:$line:1: [FILE_NAME_MATCH_CLASS] file name is incorrect - it should match with the class described in it if there is the only one class declared: Example5Expected.kt vs Some{{.*}}
 /*
-    Copyright 2018-2023 John Doe.
```

Comment:
we had similar problem with `HeaderCommentRuleFixTest`, but we introduced a placeholder `%%YEAR%%` which is replaced to current year. We should update the smoke test in same way to avoid such PRs in future